_data/toc.yaml

```diff
@@ -33,6 +33,14 @@ guides:
       path: /get-started/part2/
     - title: "Part 3: Share images on Docker Hub"
       path: /get-started/part3/
+  - sectiontitle: Node.JS
```

Comment:
Call me pedantic, but could we have lower case "JS" -> "js"? 🥰 
The normal form is `Node.js` and it looks a bit weird for Node.js developers if it's written differently.

Here is how it's written on https://nodejs.org
 
![nodejs.org homepage](https://user-images.githubusercontent.com/207759/94134084-d9f1b280-fe61-11ea-94ed-60795297be13.jpg)